pyvista/plotting/plotting.py

```diff
@@ -4095,7 +4106,7 @@ class CustomStyle(getattr(vtk, 'vtkInteractorStyle' + klass)):
 
         def __init__(self, parent):
             super().__init__()
-            self._parent = parent
+            self._parent = weakref.ref(parent)
```

Comment:
Added one more @akaszynski I found as I was working on an equivalent PR in `pyvistaqt`